electron/legendary/games.ts

```diff
@@ -275,25 +275,27 @@ class LegendaryGame extends Game {
   /**
    * Adds a desktop shortcut to $HOME/Desktop and to /usr/share/applications
    * so that the game can be opened from the start menu and the desktop folder.
-   * Both can be disabled with addDesktopShortcuts and addStartMenuShortcuts
    * @async
    * @public
    */
-  public async addShortcuts(fromMenu?: boolean) {
+  public async addShortcuts({addToStartMenu, addToDesktop}: {addToStartMenu: boolean, addToDesktop: boolean}) {
     if (process.platform === 'darwin') {
       return
     }
 
     const gameInfo = await this.getGameInfo()
     const launchWithProtocol = `heroic://launch/${gameInfo.app_name}`
-    const [ desktopFile, menuFile ] = this.shortcutFiles(gameInfo.title)
-    const { addDesktopShortcuts, addStartMenuShortcuts } = await GlobalConfig.get().getSettings()
+    const sanitizedTitle = gameInfo.title
```

Comment:
Since the issue at #760 is on Windows, this actually doesn't fix it, since this title is used only on linux.
We need another solution for windows.

Reply:
What title is used in windows to create the shortcut filename?